brainstormforce/ultimate-addons-for-gutenberg
# Fix: heading color issue on front-end in all blocks

[Trello](https://trello.com/c/bNplbEW7/189-fix-heading-color-issue)
### Description
Fixed heading color issue inside clover block.

### Block list
- Advanced Heading 
- Review Schema
- Inline notice
- Info Box
- Price list
- How to
- Faq
- Content Timeline

### Test Cases
1. Add cover block
2. inside cover block add advanced Heading block
3. apply color to the description and heading
4. Check on front-end 

### Types of changes
- Added tag to css classes at front end 

### How has this been tested?
Tested whether it is working on the front-end or not.

### Checklist:
- [ x] I've added proper labels to this pull request
